EquiCLIP/visualize_lambda.py

```diff
@@ -9,21 +9,35 @@
 import clip
 import copy
 import argparse
+import os
+from pathlib import Path
 import pytorch_lightning as pl
+import sys
 import torch.nn as nn
 import torch.optim as optim
 import torchvision
 
+# ugly way of making import work when using as a Python module
+# equiclip_path = os.path.join(os.path.dirname(__file__), "EquiCLIP")
+equiclip_path = os.path.dirname(__file__)
+assert "EquiCLIP" in equiclip_path
+if equiclip_path not in sys.path:
+    sys.path.append(equiclip_path)
+# print(sys.path)
```

Comment:
Not sure, but all hell broke loose without it.. not that I look at it, it's probably because the notebook where I imported this from was outside the EquiCLIP folder. I can move it inside and get rid of this probably

Reply:
ok, feel free not to fix it